composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "onedesign/atomic-deploy",
+    "name": "onedesign/deploy",
```

Comment:
Whoops, sorry, we want to keep this value as `onedesign/atomic-deploy` so it matches the repo